news/2022-07-08-the-roundtable.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "The Roundtable"
+date: 2022-07-08 10:00:00 +0000
+---
+
+16 osu! warriors battle in the inaugural edition of **The Roundtable**, osu!'s first major in-person invitational tournament!
+
+![](/wiki/shared/news/2022-07-08-the-roundtable/roundtable_banner.png)
+
+While LAN events are nothing new to the osu! scene, [**The Roundtable**](/wiki/Tournaments/The_Roundtable) takes a different approach.
+
+This **3 day** event from July 8-10 features **3 different tournaments**, all streamed from [BTMC](https://osu.ppy.sh/users/3171691)'s [Twitch channel](https://twitch.tv/BTMC) with comfy couch commentary and personal interactions with everyone involved!
+
+## Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| [16 player battle royale](/wiki/Tournaments/The_Roundtable/16_player_battle_royale) | 2022-07-08 (21:00 UTC +0) |
+| [2v2 team tandem tournament](/wiki/Tournaments/The_Roundtable/2v2_team_tandem_tournament) | 2022-07-09 (18:00 UTC +0) |
+| [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) | 2022-07-09 (22:00 UTC +0) |
+| [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) (top 8) | 2022-07-10 (17:30 UTC +0) |
```

Comment:
```suggestion
| [16 player battle royale](/wiki/Tournaments/The_Roundtable/16_player_battle_royale) | 2022-07-08 (21:00 UTC) |
| [2v2 team tandem tournament](/wiki/Tournaments/The_Roundtable/2v2_team_tandem_tournament) | 2022-07-09 (18:00 UTC) |
| [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) | 2022-07-09 (22:00 UTC) |
| [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) (top 8) | 2022-07-10 (17:30 UTC) |
```
also if it's important for viewers to get the right time, something like <https://github.com/ppy/osu-wiki/pull/7582/files#diff-b0dd85f0e03a4c42664ec75774aa150ed00925a33b09a06c0efe7bbf1b4b5693R106> might be helpful, they've been adding this to OWC schedules recently

Reply:
I think adding a link to show comparisons or local timezones is good, but that page used in the linked article is... useless? It just shows the UTC time for me, with ads plastered everywhere. Best to find an alternative site which actually shows the user's local timezone automatically.

Would be cool to add support for this at a wiki level at some point.